src/redteam/strategies/index.ts

```
@@ -23,6 +23,7 @@
 import { addMultilingual } from './multilingual';
 import { addOtherEncodings, EncodingType } from './otherEncodings';
 import { addPandamonium } from './pandamonium';
+import { addPlaybook } from './playbook';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'addPlaybook'. Consider to delete it or rename it.

'addPlaybook' is defined here:


(lint/suspicious/noRedeclare)